src/components/News/NewsSmallCard.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import cx from 'classnames'
+import { dateDifferenceInWords } from '../../utils/dates'
+import MultilineText from '../MultilineText/MultilineText'
+import styles from './NewsSmallCard.module.scss'
+
+const NewsSmallCard = ({ title, description, sourceName, url, datetime }) => {
+  return (
+    <a
+      href={url}
+      target='_blank'
+      rel='noopener noreferrer'
+      className={cx(styles.link)}
+    >
+      <div className={styles.news}>
+        <div className={styles.news__title}>
```

Comment:
`styles` object incapsulates CSS only for the current component. So we can safely avoid BEM to reduce the number of necessary characters.
-> `.title`, `.date` and so on